PathToRegex/PathToRegex.swift

```diff
@@ -20,7 +20,12 @@
 //===----------------------------------------------------------------------===//
 
 import Foundation
-import Regex
+
+#if COCOAPODS
+    import CrossroadRegex
```

Comment:
I think we should just release Regex 0.7 with CocoaPod Module specified as Regex to avoid all these ```if```s.